src/options.c

```diff
@@ -293,11 +295,14 @@ static void optionsParseLine(char *optarg)
             opt.lineMode = estrdup(value);
             break;
         case Opacity:
-            if (!optionsParseIsString(value)) {
-                errx(EXIT_FAILURE, "Missing value for suboption '%s'",
-                    token[Opacity]);
+            opt.lineOpacity = options_parsenum(value,
+                SELECTION_EDGE_OPACITY_MIN, SELECTION_OPACITY_MAX, &errmsg);
```

Comment:
There are 2 separate opacity ranges, one for the `scrot_selection.c` file and the other for the `selection_edge.c` file.
They must be independent.

Reply:
Hmm, I copypasted that from the original. `grep` says that in the current master, SELECTION_OPACITY_MIN and SELECTION_EDGE_OPACITY_MAX are unused. 
This seems to be because opt.lineOpacity is reused across "classic" and "edge" selection modes.

Maybe it should be left as is and fixed in a separate PR?

Reply:
I only see that `SELECTION_EDGE_OPACITY_MAX` is not used.

#### Old behavior
I will try to explain the current behavior: the highs and lows for `edge` mode is so that the selection area is **not invisible**, as it is not in `classic` mode either.
These ranges are internal to scrot and are not the responsibility of the user. The only thing the user has to know, by reading the manpage, is that the opacity range is between 0 and 255. That's  why functions like `optionsParseRequireRange` and `nonNegativeNumber` never signal an error to the user, because it's for use internal scrot.

#### New behavior

Ok, let's change the behavior of the `edge` selection mode, you have to change `SELECTION_EDGE_OPACITY_MIN` to `SELECTION_OPACITY_MIN` and remove all `_EDGE_OPACITY` references.
```suggestion
                SELECTION_OPACITY_MIN, SELECTION_OPACITY_MAX, &errmsg);
```

 This changes the old behavior of the `edge` selection mode: if the opacity is equal to 0 it draws the selection area with opacity 10 which makes it **always visible**, as it is with the `classic` selection mode, but with this change if the user uses opacity 0 the `edge` selection mode will be **invisible** (with CWM enabled of course).